beacon-chain/sync/rpc_beacon_blocks_by_range_test.go

```diff
@@ -2,6 +2,7 @@ package sync
 
 import (
 	"context"
+	"github.com/pkg/errors"
```

Comment:
Please put this import with the other non-standard imports. The second section starting on line 12